File: app/src/main/java/com/nfaralli/particleflow/WPPreferenceActivity.java
package com.nfaralli.particleflow;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;

public class WPPreferenceActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        getSupportFragmentManager()
                .beginTransaction()
                .replace(android.R.id.content, new WPPreferenceFragment())
                .commit();
    }
}

File: app/src/main/kotlin/com/kabel2/particleflow/WPPreferenceActivity.kt
package com.kabel2.particleflow

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class WPPreferenceActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        supportFragmentManager
            .beginTransaction()
            .replace(android.R.id.content, WPPreferenceFragment())
            .commit()
    }
}
